pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -822,6 +822,41 @@ def test_groupby_aggregate_period_frame(func):
     tm.assert_frame_equal(result, expected)
 
 
+def test_grouby_agg_loses_results_with_as_index_false_relabel():
```

Comment:
can you put these after the other relablng tests (IIRC in this same file)

Reply:
Done